dist/Thread-Queue/t/01_basic.t

```diff
@@ -11,16 +11,7 @@ BEGIN {
 
 use threads;
 use Thread::Queue;
-
-BEGIN { # perl RT 133382
-if ($] == 5.008) {
-    require 't/test.pl';   # Test::More work-alike for Perl 5.8.0
```

Comment:
Thread-Queue is dual-life, blead-first.  If we remove instances of `require 't/test.pl';` from the tests, are we still ready to test a CPAN release on perl-5.8?

Reply:
I don't see why not. `t/test.pl` only exists in the perl core. Anything installed from CPAN won't have access to it anyway.

Besides, the version check looks for 5.8.0 exactly. Anything else, including perl 5.8.1, 5.8.2, 5.8.3, etc, has always used the Test::More branch of the conditional.